crates/key-server/src/server.rs

```diff
@@ -255,7 +257,18 @@ impl Server {
             .await
             .map_err(|e| {
                 warn!("Dry run execution failed ({:?}) (req_id: {:?})", e, req_id);
```

Comment:
warn! should be called only for non user errors (thus not for INVALID_PARAMS_CODE), otherwise it should be debug!